server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/OrganizationTester.java

```diff
@@ -117,15 +120,15 @@ public OrganizationTester assertThatNotMemberOf(Organizations.Organization organ
 
   private void verifyOrganizationMembership(@Nullable Organizations.Organization organization, String userLogin, boolean isMember) {
     List<Organizations.User> users = service().searchMembers(new SearchMembersRequest()
-      .setQuery(userLogin)
+      .setQ(userLogin)
       .setSelected("selected")
       .setOrganization(organization != null ? organization.getKey() : null))
       .getUsersList();
     Assertions.assertThat(users).hasSize(isMember ? 1 : 0);
   }
 
   private void verifyMembersGroupMembership(String userLogin, @Nullable Organizations.Organization organization, boolean isMember) {
-    List<Users.GroupsWsResponse.Group> groups = session.wsClient().users().groups(GroupsRequest.builder()
+    List<Users.GroupsWsResponse.Group> groups = session.wsClient().usersOld().groups(GroupsRequest.builder()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "usersOld"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)